bin/start-local.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Function to clean up processes and ports
+cleanup() {
+    echo "Cleaning up..."
+    # Kill the SAM process if it's running
+    if [ ! -z "$SAM_PID" ]; then
+        kill $SAM_PID 2>/dev/null
+    fi
+    # Kill any process using port 3013
+    lsof -ti:3013 | xargs kill -9 2>/dev/null
+    exit 0
+}
+
+# Set up trap to call cleanup function on Ctrl+C
+trap cleanup SIGINT
+
+# Stop, start and setup RDF4J
+echo "Starting RDF4J..."
+export RDF4J_USER_NAME=rdf4j
+export RDF4J_PASSWORD=rdf4j
+npm run rdf4j:stop
+npm run rdf4j:start
+# Wait for rdf4j server to start
+sleep 10
+npm run rdf4j:setup
```

Comment:
I don't think starting the db should be part of start-local.   You might want to start-local multiple times and keep the data in the rdfdb.